.formatter.exs

```diff
@@ -1,4 +1,4 @@
-  # Used by "mix format"
```

Comment:
Can you remove this file from the PR? Its outside the scope of what you're requesting to change